### PR TITLE
Add WNP 121 

### PR DESCRIPTION
**Pull request by reemhamz**
## One-line summary
Adding WNP121. The same template will be used for all locales! And **no variations** ❗ 💯 

We have `en-US`, `en-GB`, `fr`, and `de` translations hard-coded into the template.

- [Copy](https://docs.google.com/document/d/1C-DbPPmue_S7YbWTzETOkk4DfSNM-HFEl9uaDCxhCio/edit)
- [Figma design](https://www.figma.com/file/CZvzHgU9NQMQUm34lomIl5/Firefox-WNP?type=design&node-id=5474%3A1322&mode=design&t=HaDhAXmSSMYm2ylh-1)

> [!IMPORTANT]
> Make sure the links inside the boxes have the correct geo mentioned in the `utm_campaign`

## Issue / Bugzilla link
https://app.asana.com/0/home/1202716623574806/1206078601386171

## Testing
- http://localhost:8000/en-US/firefox/121.0/whatsnew/
- https://www-demo6.allizom.org/en-US/firefox/121.0/whatsnew/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx120-eu.html [hunk 1]
```diff
@@ -56,7 +56,6 @@ <h2 class="wnp-main-title">{{ headline }}</h2>
 
         <button class="wnp-main-cta mzp-c-button mzp-t-product uitour">{{ cta_text }}</button>
         <a href="https://support.mozilla.org/kb/customize-firefox-controls-buttons-and-toolbars" rel="external noopener" class="wnp-main-cta mzp-c-button mzp-t-product sumo-link">{{ cta_text }}</a>
-      </div>
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx120-eu.html:59]: Realized there was a missing div on this template

**Review by craigcook — CHANGES_REQUESTED** [review]
Nice work! ✨  Just a few small nits and we can ship this baby before code freeze.

File: bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx121.html [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  {% set box_two_headline = 'Set up Firefox </br> in a flash' | safe %}
+  {% set box_two_lede = 'Easily back-up your Firefox tabs and history, and instantly access on your other devices.' %}
+  {% set box_two_cta = 'Meet your back up assistant' %}
+  {% set box_two_link = 'https://support.mozilla.org/kb/switching-devices' + utm_params_backup_na %}
+  {% set sign_off = ftl('whatsnew-signoff') %}
+{% endif %}
+
+{% block wnp_content %}
+  <section class="wnp-content-main">
+    <div class="mzp-l-content mzp-t-content-lg">
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx121.html:95]: Agh, good looking out

File: media/css/firefox/whatsnew/whatsnew-121.scss [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+            }
+
+            .mzp-c-cta-link {
+                @include font-base;
+            }
+        }
+
+        .c-box-image {
+            margin-top: $spacing-lg;
+            }
```
[the remark this answers is not in the document]
Comment (reemhamz) [reply media/css/firefox/whatsnew/whatsnew-121.scss]: 🦅 👁️

**Review by craigcook — APPROVED** [review]

**Pull request merged**